spec/helpers/params_spec.rb

```diff
@@ -34,4 +34,18 @@ def app
     post "/", "<hello>world</hello>", {"CONTENT_TYPE" => "application/xml"}
     assert_equal "{}", last_response.body
   end
+
+  it 'should throw bad request when receiving invalid json via post' do
+    assert_raises(Pliny::Errors::BadRequest) do
+      post '/', '{\"foo\"}', {"CONTENT_TYPE" => "application/json"}
+    end
+  end
+
+  it 'should throw bad request and the correct message when post has invalid json' do
+    begin
+      post '/', '{\"foo\"}', {"CONTENT_TYPE" => "application/json"}
```

Comment:
For consistency, could we change these to use double-quotes?

```suggestion
  it "should throw bad request when receiving invalid json via post" do
    assert_raises(Pliny::Errors::BadRequest) do
      post "/", "{\"foo\"}", {"CONTENT_TYPE" => "application/json"}
    end
  end

  it "should throw bad request and the correct message when post has invalid json" do
    begin
      post "/", "{\"foo\"}", {"CONTENT_TYPE" => "application/json"}
```

Reply:
Yeah no worries. My rubocop was defaulting back to "single quote style unless necessary".